src/components/tasks/card/card.module.scss

```diff
@@ -58,3 +58,40 @@
   border: solid red 2px;
   background-color: #ffebee;
 }
+
+.progressContainer {
+  width: 13rem;
+  flex-direction: column;
+}
+
+.progressIndicator {
+  width: 100%;
+  height: 0.6rem;
+  background-color: #e5e5e5;
+  border: 1.5px solid #252525;
+  border-radius: 1rem;
+}
+
+.progressGreen {
+  background-color: green;
+  height: .4rem;
+  border-radius: 1rem;
+}
+
+.progressOrange {
+  background-color: orange;
```

Comment:
use these colors as css variables something like `--progress-orange` what this does is it will help you get your work done faster plus you replace color in one place and gets affected in multiple places/files, 
and if you create these as classes it will help more something like where you have height & border-radius as same since its repeating in progressRed, progressOrange & etc

Reply:
Thanks vaibhav, I will do it